Core/Src/main.c

```diff
@@ -98,7 +119,22 @@ int main(void)
     /* USER CODE END WHILE */
 
     /* USER CODE BEGIN 3 */
+
+	/* Initiate Tx-Rx*/
+	HAL_GPIO_WritePin(GPIOB, GPIO_PIN_8, GPIO_PIN_RESET);
+	HAL_SPI_TransmitReceive(&hspi1, tx_data, rx_data, 3, 100);
```

Comment:
This looks good, but in terms of coding practices, it would be nice to have macros or constants for the `3` and `100` to make it more readable, bec to someone just looking at it, they may have no idea what that is referring to